engine/Controls.js

```diff
@@ -78,10 +80,8 @@ function moveRotateAndSprint(dx, dy, sprinting , keyPressedFlag) {
 
     // Update the position of plane based on the direction of movement and sprint status , only if any of the keys is pressed
     if (keyPressedFlag) {
-        posX += dx * speed * 10 + 30 * dx * t;
```

Comment:
removing the `* t` in these two lines seems to have made the speed of the plane by default incredibly fast.

Reply:
Hello! I've fixed the bug by adding t back.